MainRepository/Program.cs

```diff
@@ -1,7 +1,68 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Program to find the possible combination of queen in the n*n chess board
+class NQueen {
+   static int Count;
+   //Function to check if two queens attack each other or not

```

Comment:
```suggestion
   // Function to check if two queens attack each other or not

```
Space after // required